modules/kubernetes-addons/adot-collector-nginx/otel-config/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-prometheus-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: otel-prometheus-role
+subjects:
+  - kind: ServiceAccount
+    name: adot-collector
+    namespace: adot-collector
```

Comment:
`adot-collector-nginx`, also add a newline after to pass linter checks.

Reply:
modified